code/modules/mob/inventory.dm

```diff
@@ -128,15 +128,25 @@
 	Arguments
 	* obj/item/W is the item you are trying to equip
 	* del_on_fail if true will delete the item instead of dropping it to the floor
+	* merge_stacks if true will merge stacks that src is holding in its hands with new ones
 
 	Returns TRUE if it was able to put the thing into one of our hands.
 */
-/mob/proc/put_in_hands(obj/item/W, del_on_fail = FALSE)
+/mob/proc/put_in_hands(obj/item/W, del_on_fail = FALSE, merge_stacks = TRUE)
 	W.do_pickup_animation(src)
 	if(status_flags & INCORPOREAL) // INCORPOREAL things don't have hands
 		return FALSE
 	if(!W)
 		return FALSE
+	if(merge_stacks && istype(W, /obj/item/stack))
+		var/obj/item/stack/S = W
+		var/obj/item/stack/D = null
```

Comment:
Don't use single letter var names like this

Reply:
I thought this is the code style because I saw that everywhere, I'll fix it